documentation/modules/mtv-performance-recommendation.adoc

```diff
@@ -174,4 +170,43 @@ Testing showed that by breaking one single large plan into multiple moderately s
 * *Maximum cold migration transfer rate of the multiple VMs (single disk) from a single ESXi host: 294 MiB/s (concurrent migration of 30 VMs, 35/50 GiB used, from Single ESXi)*
 * *Maximum cold migration transfer rate of the multiple VMs (single disk) from multiple ESXi hosts: 1173MB/s (concurrent migration of 80 VMs, 35/50 GiB used, from 8 ESXi servers, 10 VMs from each ESXi)*
 
-For additional details on performance, see xref:mtv-performance-addendum_mtv[{project-short} performance addendum]
+[id="mtv-warm-migration-recommendations_{context}"]
+== Warm migration recommendations
+
+The following are recommendations that are specific to warm migrations:
+
+[id="mtv-warm-migration-recommendations-disks-in-parallel_{context}"]
+=== Migrate up to 400 disks in parallel
+
+Testing involved migrating 200 VMs, with 2 disks each using 8 ESX hosts. All VMs were distributed evenly over the ESX hosts, 25 VMs per ESX host. During the tests, the ESX hosts monitored the total connections of `DataTransfer`. The results show each ESX host had 50 open connections for disk transfer.
```

Comment:
I feel the wording here is less customer-friendly. Currently, it sounds like a test report. Please consider rewording.something like " Testing involved migrating 200 VMs, with 2 disks each using 8 ESX hosts. All VMs were distributed evenly over the ESX hosts, 25 VMs per ESX host - 400 disks TotallAnd say that we cant recommend going higher than what we tested... we don't know what will be the impact.
 Remove this - to technical: "During the tests, the ESX hosts monitored the total connections of DataTransfer. The results show each ESX host had 50 open connections for disk transfer."  @duduvaa  what do you think